### PR TITLE
Add initlog to grid tests

### PR DESCRIPTION
**Pull request by drwells**
Part of #4250.

### CODE REVIEW

File: include/deal.II/base/logstream.h [hunk 1]
```diff
@@ -144,10 +144,16 @@ class LogStream : public Subscriptor
   /**
    * Enable output to a second stream <tt>o</tt>.
    *
-   * The optional argument @p print_job_id specifies whether
+   * @p print_job_id Whether or not the JobIdentifier for the current process
```
Comment (bangerth) [inline include/deal.II/base/logstream.h]: You want `@param[in]` here. Same below.

File: include/deal.II/base/logstream.h [hunk 2]
```diff
@@ -144,10 +144,16 @@ class LogStream : public Subscriptor
...
+   * should be printed to the stream.
+   *
+   * @p flags Format flags to set on the output stream @p o.
    */
   void
-  attach(std::ostream &o, const bool print_job_id = true);
+  attach(std::ostream &                o,
+         const bool                    print_job_id = true,
+         const std::ios_base::fmtflags flags        = std::ios::showpoint |
+                                               std::ios::left);
```
Comment (bangerth) [inline include/deal.II/base/logstream.h]: oh, indentation fail :-(

File: include/deal.II/grid/tria.h [hunk 3]
```diff
@@ -2823,6 +2823,27 @@ class Triangulation : public Subscriptor
...
+   *     face->set_manifold_id(42);
+   * @endcode
+   *
+   * @return The half open range <code>[this->begin_active_face(),
+   * this->end_face())</code>
+   *
+   * @ingroup CPP11
+   */
+  IteratorRange<active_face_iterator>
+  active_face_iterators() const;
```
Comment (bangerth) [inline include/deal.II/grid/tria.h]: did you mean to leave this for another PR?

Comment (drwells) [reply include/deal.II/grid/tria.h]: whoops. How did that get there?

File: include/deal.II/base/logstream.h (hunk 2 above)
Comment (drwells) [reply include/deal.II/base/logstream.h]: That one is not so pretty.

**Comment by drwells** [comment]
I fixed the doxygen labeling (i.e., use `@param[in]`).
